sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-messaging/src/main/java/com/azure/spring/sample/messaging/WebController.java

```diff
@@ -17,8 +17,8 @@
 @RestController
 public class WebController {
 
-    private static final String EVENT_HUB_NAME = "eventhub1";
-    private static final String CONSUMER_GROUP = "cg1";
+    private static final String EVENT_HUB_NAME = "event-0329-hub";
```

Comment:
will "event-0329-hub" be created automaticly or need create before configuration

Reply:
do we need to write something about this in readme.md?

Reply:
Yes i agree